cookbooks/aws-parallelcluster-environment/resources/cloudwatch/partial/_cloudwatch_common.rb

```diff
@@ -137,7 +137,7 @@ def package_path
     path common_module_path
     user 'root'
     group 'root'
-    mode '0644'
+    mode '0755'
```

Comment:
Just out of my curiosity: in the first version of this change the script was not executable by root, so how did you test the change if the script is executed by root on node bootstrap?

Reply:
I ran the kitchen tests, logged in to the image, and ran whoami to check the permissions.

Reply:
> whoami to check the permissions.

You mean to check the current user?

> I ran the kitchen tests

Do kitchen tests verify that the generate cwagent config is as expected?

Reply:
I am wondering if we should change the Permission. The permissions given to the new common utils module is similar to the permissions given to the other python modules, how is there no issue when existing python modules have same non-executable permissions.